tests/inputs/example_service/example_service.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package example_service;
+
+service ExampleService {
```

Comment:
Changed it. Now that I think about it, `example_service` might not be too good a test case name since it's supposed to be an actual test instead of just something for the README, like the `example` case.

What do you think about integrating this test into the existing `service` test case? It seems pretty similar.